github/interactions.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+// InteractionsService handles communication with the repository and organization related
+// methods of the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/interactions/
+type InteractionsService service
+
+// Interaction represents the interaction restrictions for repository and organization.
+type Interaction struct {
```

Comment:
Should this be `InteractionRestriction`? 

`Interaction` with these fields wouldn't read particularly in my head if I was working with docs / autocomplete.

Reply:
Excellent point, @gauntface! I agree. Let's please change this, @waseem18.

Reply:
Addressed in a new commit - https://github.com/google/go-github/pull/1079/commits/c93e1de5d098a64104ff0c9d4933bc92e87e3388 👍